Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/VJBK-Jiva-replica-failure/jiva-replica-failure

```diff
@@ -0,0 +1,176 @@
+echo $test_name
+
+################
+# LitmusBook 1 #
+################
+
+cd litmus
+echo "Running the litmus test for Busybox Deployment.."
+cp apps/busybox/deployers/run_litmus_test.yml deploy_run_test_jiva.yml
+
+: << EOF
+  ---------------------------------------------------------------------------------------------------------------------
+ | specAttribute     | kind   |         default value               | test specifc value                               |
+  ---------------------------------------------------------------------------------------------------------------------|
+ | appLabel          | env    | app=busybox-sts                     | app=busybox-rep-fail-jiva                        |
+ | appNamespace      | env    | app-busybox-ns                      | app-replica-fail-jiva                            | 
+ | appPVC            | env    | openebs-busybox                     | openebs-busybox-rep-kill                         |
+ | storage class     | env    | openebs-cstor-sparse                | openebs-jiva-default                             |
+ | deploy type       | env    | statefulset                         | deployment                                       |
+ | Action            | env    | value: provision                    | value: deprovision                               |
+ | Litmus Job name   | name   | generateName: litmus-busybox-deploy | generateName: busybox-provision-app-rep-failure  |
+ | Litmus job label  | label  | app: busybox-litmus                 | app: busybox-replica-failure-jiva                |
+  ----------------------------------------------------------------------------------------------------------------------
+EOF
+
+sed -i -e 's/imagePullPolicy: Always/imagePullPolicy: IfNotPresent/g' \
+-e 's/generateName: litmus-busybox-deploy/generateName: busybox-provision-app-rep-failure/g' \
+-e 's/value: app-busybox-ns/value: app-replica-fail-jiva/g' \
+-e 's/value: openebs-busybox/value: openebs-busybox-rep-kill/g' \
+-e 's/app: busybox-litmus/app: busybox-replica-scale-jiva/g' \
+-e 's/value: '\''app=busybox-sts'\''/value: '\''app=busybox-rep-fail-jiva'\''/g' \
+-e 's/value: openebs-cstor-sparse/value: openebs-jiva-default/g' deploy_run_test_jiva.yml
+
+cat deploy_run_test_jiva.yml
+
+bash ../Openshift-EE/utils/litmus_job_runner label='app:busybox-replica-failure-jiva' job=deploy_run_test_jiva.yml
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:app-replica-failure-jiva $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id VJBK 4-chaos "Jiva Replica Failure" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+################
+# LitmusBook 2 #
+################
+
+run_id="jiva";test_name=$(bash Openshift-EE/utils/generate_test_name testcase=volume-replica-failure metadata=${run_id})
```

Comment:
@shashank855 this test case name must be the same as there in the test_vars.
`https://github.com/openebs/litmus/blob/master/apps/percona/chaos/openebs_volume_replica_failure/test_vars.yml`